sympy/geometry/entity.py

```diff
@@ -69,6 +69,8 @@ class GeometryEntity(Basic, EvalfMixin):
 
     """
 
+    __slots__ = ()
```

Comment:
The mypy tests are failing because mypy can't figure out the "type" of slots here. I think it needs something like this to make mypy happy:
```diff
diff --git a/sympy/geometry/entity.py b/sympy/geometry/entity.py
index 1bd1722..5d541c1 100644
--- a/sympy/geometry/entity.py
+++ b/sympy/geometry/entity.py
@@ -19,6 +19,7 @@
 R3 are currently the only ambient spaces implemented.
 
 """
+from typing import Tuple as tTuple
 
 from sympy.core.basic import Basic
 from sympy.core.containers import Tuple
@@ -69,7 +70,7 @@ class GeometryEntity(Basic, EvalfMixin):
 
     """
 
-    __slots__ = ()
+    __slots__: tTuple[()] = ()
 
     def __cmp__(self, other):
         """Comparison of two GeometryEntities."""
```

Reply:
That'd make `mypy` happy, although I'd probably go for `tTuple[str, ...]` to ensure slots may be added later.